angular/src/app/app.module.ts

```diff
@@ -21,10 +25,12 @@ import { SampleService } from "./modules/sample/sample.service";
     BrowserModule,
     FormsModule,
     HttpClientModule,
+    HttpModule,
     routing,
-    SampleModule
+    SampleModule,
+    DevicesModule
   ],
-  providers: [appRoutingProviders, SampleService],
+  providers: [appRoutingProviders, SampleService, DevicesService],
```

Comment:
Can you put the DevicesService in the providers list inside "Devices" module instead? I decided to not bloat appModule every time we added a feature, but I haven't made changes to SampleService